src/models/Settings.js

```diff
@@ -66,7 +66,6 @@ export default Model.extend({
     'features.webauthn': ['boolean', true, false],
     'features.selfServiceUnlock': ['boolean', true, false],
     'features.multiOptionalFactorEnroll': ['boolean', true, false],
-    'features.preventBrowserFromSavingOktaPassword': ['boolean', true, true],
```

Comment:
was this change intentional? does not seem to belong to this PR

Reply:
yeah, I shall mentioned in the description. I meant for 2 commits to fix those 2 problems.